docs/docs-content/automation/automation.md

```diff
@@ -16,14 +16,18 @@ This section contains documentation and guides for tools essential in automating
   [self-hosted Palette](../enterprise-version/enterprise-version.md) instance and deploying a
   [Private Cloud Gateway](../clusters/pcg/pcg.md).
 
+- Palette Go SDK - Provides client configuration through a set of public libraries, enabling Go developers to interact
```

Comment:
suggestion:

Palette Go SDK - Enables developers to interact with Palette APIs for automated resource management using Go.